modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-status.adoc

```diff
@@ -38,226 +60,260 @@ rpk cluster self-test status [flags]
-            191us  447us  831us  2175us  278527us
-
-NAME        8K Network Throughput Test
-INFO        Test performed against node: 0
-TYPE        network
-TEST ID     5e4052f3-b828-4c0d-8fd0-b34ff0b6c35d
-TIMEOUTS    0
-DURATION    5000ms
-IOPS        61254 req/sec
-THROUGHPUT  3.74Gib/sec
-LATENCY     P50    P90    P99    P999   MAX
-            159us  207us  303us  415us  1087us
-
-NAME        8K Network Throughput Test
-INFO        Test performed against node: 2
-TYPE        network
-TEST ID     5e4052f3-b828-4c0d-8fd0-b34ff0b6c35d
-TIMEOUTS    0
-DURATION    5000ms
-IOPS        54814 req/sec
-THROUGHPUT  3.35Gib/sec
-LATENCY     P50    P90    P99    P999   MAX
-            167us  255us  367us  511us  25599us
+NAME          512KB sequential r/w throughput disk test
+INFO          write run
+TYPE          disk
+TEST ID       5632cefe-cf42-44ab-bba9-18fa1c71e3ee
+TIMEOUTS      0
+START TIME    Thu Jul 11 18:58:13 UTC 2024
+END TIME      Thu Jul 11 18:58:43 UTC 2024
+AVG DURATION  30007ms
+IOPS          140 req/sec
+THROUGHPUT    70.17MiB/sec
+LATENCY       P50      P90      P99       P999      MAX
+              24575us  47103us  188415us  425983us  507903us
+
+NAME          512KB sequential r/w throughput disk test
+INFO          read run
+TYPE          disk
+TEST ID       5632cefe-cf42-44ab-bba9-18fa1c71e3ee
+TIMEOUTS      0
+START TIME    Thu Jul 11 18:58:43 UTC 2024
+END TIME      Thu Jul 11 18:59:13 UTC 2024
+AVG DURATION  30008ms
+IOPS          276 req/sec
+THROUGHPUT    138.5MiB/sec
+LATENCY       P50      P90      P99      P999     MAX
+              13823us  19455us  24575us  57343us  77823us
+
+NAME          4KB sequential r/w latency/iops disk test
+INFO          write run
+TYPE          disk
+TEST ID       5632cefe-cf42-44ab-bba9-18fa1c71e3ee
+TIMEOUTS      0
+START TIME    Thu Jul 11 18:59:13 UTC 2024
+END TIME      Thu Jul 11 18:59:43 UTC 2024
+AVG DURATION  30000ms
+IOPS          6769 req/sec
+THROUGHPUT    26.44MiB/sec
+LATENCY       P50    P90    P99     P999     MAX
+              191us  255us  2303us  13311us  81919us
+
+NAME          4KB sequential r/w latency/iops disk test
+INFO          read run
+TYPE          disk
+TEST ID       5632cefe-cf42-44ab-bba9-18fa1c71e3ee
+TIMEOUTS      0
+START TIME    Thu Jul 11 18:59:43 UTC 2024
+END TIME      Thu Jul 11 19:00:13 UTC 2024
+AVG DURATION  30001ms
+IOPS          13235 req/sec
+THROUGHPUT    51.7MiB/sec
+LATENCY       P50    P90    P99    P999    MAX
+              127us  239us  735us  1919us  63487us
+
+NAME          8Kb Network Throughput Test
+INFO          Test performed against node: 2
+TYPE          network
+TEST ID       5632cefe-cf42-44ab-bba9-18fa1c71e3ee
+TIMEOUTS      0
+START TIME    Thu Jul 11 19:00:13 UTC 2024
+END TIME      Thu Jul 11 19:00:43 UTC 2024
+AVG DURATION  30000ms
+IOPS          55370 req/sec
+THROUGHPUT    3.38Gib/sec
+LATENCY       P50    P90    P99    P999   MAX
+              167us  231us  351us  495us  7679us
+
+NAME          Cloud Storage Test
```

Comment:
Output of `Cloud Storage Test` here (and throughout) doesn't reflect the reductions made in https://github.com/redpanda-data/redpanda/pull/18787.

Reply:
Updated!